<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>motionCardWidget</class>
 <widget class="QWidget" name="motionCardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>541</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>运动控制卡设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>变量结果</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="confirmButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>测试</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goodButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>良品</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="badButton">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>不良品</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
